cf-usb/tasks/service-build.yml

```diff
@@ -15,6 +15,6 @@ outputs:
 - name: helm-out
 - name: docker-out
 run:
-  args:
-  - mysql
   path: ci/cf-usb/tasks/service-build.sh
+params:
+  SERVICE: ~
```

Comment:
Does it need this ? Would it need `DESTINATION` as well ?

Reply:
Nah, it actually needs neither (because they get overridden from the pipeline). I'm just putting it here as a reference for people looking at reusing the task next time (e.g. for a redis service).

Reply:
I see. Ok.